src/components/GraphHeartbeats.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Paper } from 'material-ui'
+import Divider from 'material-ui/Divider'
+import { BarChart, Bar, Cell } from 'recharts'
+import Theme from '../../style/theme'
+
+class GraphHeartbeats extends React.PureComponent {
+  static getColor(amt) {
+    if (amt > 1) {
+      return '#00E200'
```

Comment:
Could you use colors from Colors.js? Those are all the material colors. If you still prefer these, just throw these colors in that file and use it from there. Having colors, styles, etc as variables when possible make it easier to swap and replace especially if stuff gets reused across multiple components.

Reply:
I think we should use material colors wherever possible. Whatever their default success and error colors are.

Reply:
I'm going to intentionally leave the hardcoded '#00E200' because the rest of this function is returning a function somewhere half midway between '#00E200' and '#E20000'. This method is left intentionally undocumented because I want it to eventually get rewritten in a better way.